users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from core.models import TimeStampModel
+
+class User(TimeStampModel):
+    first_name   = models.CharField(max_length=100)
+    last_name    = models.CharField(max_length=100)
+    email        = models.CharField(max_length=200, unique=True)
+    phone_number = models.CharField(max_length=200)
+    birthday     = models.DateField()
+    gender       = models.CharField(max_length=100)
+    kakao        = models.BigIntegerField()
```

Comment:
1. kakao 보다는 kakao_id를 더 많이 사용합니다.
2. kakao로 회원가입을 진행 할 때 못 받는 값들에 대해서는 null=True 또는 default 값을 주시면 좋을 것 같습니다.
```suggestion
    kakao_id      = models.BigIntegerField()
```